src/devices/uFire_ISE/UFirePh .cs

```diff
@@ -4,95 +4,101 @@
 
 using System;
 using System.Device.I2c;
+using UnitsNet;
 
 namespace Iot.Device.UFire
 {
     /// <summary>
     /// Get pH measuremens from μFire ISE Probe Interface
     /// </summary>
-    public class UFire_pH : UFire_ISE
+    public class UFirePh : UFireIse
     {
         private const float PROBE_MV_TO_PH = 59.2F;
         private const float TEMP_CORRECTION_FACTOR = 0.03F;
 
         /// <summary>
-        /// pH measuremen
+        /// pH (Power of Hydrogen) units measurement
         /// </summary>
-        public float PH = 0;
+        public float Ph = 0;
 
         /// <summary>
-        /// poH measuremen
+        /// pOH units measurement
```

Comment:
what is pOH vs pH, please add in the comments. Also, like for the rest of similar comments, consider using the full name rather than the abbreviation

Reply:
pOH is a unit and I can not write it in any other way then pOH. I have added a comment explaining the relationship between pH and pOH